core/domain/app_feedback_report_domain.py

```diff
@@ -736,13 +736,13 @@ def require_valid_version_name(cls, version_name: str) -> None:
         Raises:
             ValidationError. The given app version name is not valid.
         """
+        delimiter = app_feedback_report_constants.ANDROID_VERSION_NAME_DELIMITER
```

Comment:
Declare this just above the place you use it, so that the reader doesn't have to carry along the context in their head.

Reply:
Done